sdk/python/kfp_tekton/tekton.py

```diff
@@ -20,16 +20,15 @@
 
 
 class AnySequencer(ContainerOp):
-    """Represents an op present that will proceed when any denpendent condition met
+    """A containerOp that will proceed when any dependent condition met
```

Comment:
maybe ..." _if any of the dependent containerOps completed successfully_"

Reply:
done